firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java

```diff
@@ -255,6 +252,22 @@ private static FirebaseAppDistributionException getExceptionForHttpResponse(
     }
   }
 
+  private static FirebaseAppDistributionException getExceptionFor403(
+      String tag, String responseBody) {
+    // Check if this is an API disabled error
+    TesterApiDisabledErrorDetails apiDisabledErrorDetails =
+        TesterApiDisabledErrorDetails.tryParse(responseBody);
+    if (apiDisabledErrorDetails != null) {
+      String messageWithHelpLinks =
+          String.format(
+              "%s\n\n%s", ErrorMessages.API_DISABLED, apiDisabledErrorDetails.formatLinks());
+      return getException(tag, messageWithHelpLinks, Status.API_DISABLED);
+    }
+
```

Comment:
Would it be worth logging the responseBody in this case (for example, so the developer could print the logs in `--verbose` or `--debug` somehow?

Reply:
It would! We actually print the response body out in verbose mode already: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java#L183